### ISSUE TITLE
Make project compatible with server-side render

### ISSUE THREAD

**Issue by 2fd** [decentraland/ui#53]
## Motivation:
We are adopting [`gatsbyjs`](https://www.gatsbyjs.org/) as our framework to generate static sites and it will be a goal if we could use this project because:
- it will help maintain a consistent experience through sites
- it will be easier and faster develop new sites

## Curren limitations:
The only restriction to make a project compatible with server-side render with [`gatsbyjs`](https://www.gatsbyjs.org/) is never access the Web APIs (DOM, Window, Storage, etc) outside of `componentDidMount` or `useEffect`

Currently only [`src/components/Atlas/Atlas.tsx`](https://github.com/decentraland/ui/blob/master/src/components/Atlas/Atlas.tsx) isn't compatible with SSR (because a dependency). To avoid a compilation error we must be able to import each component individually.

### Changes required

- [x] Avoid index import

```diff
-    import { Locale } from '../..'
+    import { Locale } from '../LanguageIcon/LanguageIcon.tsx'
```
> https://github.com/decentraland/ui/blob/master/src/components/Footer/Footer.tsx#L8

- [x] Generate individual files for each component, so we can import as follow

```typescript
    import { Footer } from 'decentraland-ui/lib/components/Footer'
```

- [x] Generate files for css-only components
    - './components/Button/Button.css'
    -  './components/Container/Container.css'
    -  './components/Card/Card.css'
    -  './components/Dimmer/Dimmer.css'
    -  './components/Dropdown/Dropdown.css'
    -  './components/Header/Header.css'
    -  './components/HeaderMenu/HeaderMenu.css'
    -  './components/Modal/Modal.css'
    -  './components/Loader/Loader.css'
    -  './components/Pagination/Pagination.css'
    -  './components/Popup/Popup.css'
    -  './components/Radio/Radio.css'
    -  './components/Segment/Segment.css'
    -  './components/Table/Table.css'

```diff
# ./components/Button/Button.css
+ export { Button } from 'semantic-ui-react'
+ import './Button.css'
```

### Others improvements:
- [x] avoid `componentWillMount`: is deprecated and it will be removed in next major [more](https://reactjs.org/docs/react-component.html#unsafe_componentwillmount)
- [x] move [`Locale`](https://github.com/decentraland/ui/blob/master/src/components/LanguageIcon/LanguageIcon.tsx#L4) definition outside `LanguageIcon`
- [x] move [`react-tile-map`](https://github.com/decentraland/ui/blob/master/src/index.ts#L8) inside `Atlas.tsx`

**Comment by cazala** [comment]
Closed  by #60

**Comment by brennanfife** [comment]
SSR works with v2.15.0, but at some point, it became incompatible (even before v3.0.0)
